<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>600</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Выберете имя группы:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_button">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>10</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;
border-color: rgb(220, 220, 220);
border-radius: 8px;</string>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>10</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;
border-color: rgb(220, 220, 220);
border-radius: 8px;
</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QComboBox" name="choose_comboBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>370</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe UI&quot;;
background-color: rgb(255, 255, 255);
</string>
   </property>
  </widget>
  <widget class="QTableView" name="init_table">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>64</y>
     <width>1000</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
